Seed.Api.IntegrationTests/Controllers/UserIntegrationTest.cs

```diff
@@ -0,0 +1,285 @@
+using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
+using Seed.Api.IntegrationTests.Controllers.TestData;
+using Seed.Api.IntegrationTests.Generics;
+using Seed.Api.Models;
+using Seed.Data.EF;
+using Seed.Data.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Seed.Api.IntegrationTests.Controllers
+{
+    public class UserIntegrationTest : IClassFixture<ApiTestFixture>
+    {
+        private const string ResouceUri = "api/users/";
+
+        private readonly HttpClient _client;
```

Comment:
I like how "client" reads because is very short (so it's very attractive). But I do recognize too that "client" is a broad word and specially in our field, can mean very different things. In order to shortcut that thinking when someone reads it, I think `_httpClient` would be easier to quickly identify.

Reply:
god bless crtl +r, crtl +r